docs/Makefile

```diff
@@ -18,6 +18,9 @@ publish: clean
 	$(MAKE) build
 	$(MAKE) -C deploy publish
 
+live:
+	npx nodemon --ext txt,js,html --exec "$(MAKE) -j build" --ignore site
```

Comment:
blocking: remove the `-j`

this is not parallelizable, so let's remove the `-j`

Reply:
removed it. was trying to make it future proof, if that ever happens : )